diff --git a/app/main.main.ts b/app/main.main.ts
index fe3a4b9a8..2d10750e5 100644
--- a/app/main.main.ts
+++ b/app/main.main.ts
@@ -2639,9 +2639,16 @@ app.on(
   }
 );
 
-app.setAsDefaultProtocolClient('sgnl');
-app.setAsDefaultProtocolClient('signalcaptcha');
-
+if (!app.isDefaultProtocolClient('sgnl')) {
+  app.setAsDefaultProtocolClient('sgnl');
+} else {
+  log.info('App is already registered as the default app for the sgnl url scheme.');
+}
+if (!app.isDefaultProtocolClient('signalcaptcha')) {
+  app.setAsDefaultProtocolClient('signalcaptcha');
+} else {
+  log.info('App is already registered as the default app for the signalcaptcha url scheme.');
+}
 ipc.on(
   'set-badge',
   (_event: Electron.Event, badge: number | 'marked-unread') => {
