diff --git a/node_modules/fs-extra/lib/ensure/file.js b/node_modules/fs-extra/lib/ensure/file.js
index 67eed30..7b115a2 100644
--- a/node_modules/fs-extra/lib/ensure/file.js
+++ b/node_modules/fs-extra/lib/ensure/file.js
@@ -8,8 +8,14 @@ const pathExists = require('../path-exists').pathExists
 
 function createFile (file, callback) {
   function makeFile () {
+    console.log(`Going to create empty file/dir ${file}`)
     fs.writeFile(file, '', err => {
-      if (err) return callback(err)
+      if (err && process.platform === 'linux' && process.arch === 'arm64') {
+        console.log(`Error while creating empty file, ignoring: ${err}`)
+      } else if (err) {
+        console.log(err)
+        return callback(err)
+      }
       callback()
     })
   }