diff --git a/lib/ensure/file.js b/lib/ensure/file.js
index a55c2d96351919ea10da7e388bf9e98637203da2..448091a164be974696a02259aa9ee2bc8e605474 100644
--- a/lib/ensure/file.js
+++ b/lib/ensure/file.js
@@ -5,6 +5,19 @@ const path = require('path')
 const fs = require('../fs')
 const mkdir = require('../mkdirs')
 
+async function makeFile (file) {
+  try {
+    console.log(`Going to create empty file ${file}`)
+    await fs.writeFile(file, '')
+  } catch (err) {
+    if (process.platform === 'linux' && process.arch === 'arm64') {
+      console.log(`Error while creating empty file, but ignoring since this is Linux arm64 and the file has most likely been created just fine: ${err}`)
+    } else {
+      throw err
+    }
+  }
+}
+
 async function createFile (file) {
   let stats
   try {
@@ -21,7 +34,7 @@ async function createFile (file) {
     // if the directory doesn't exist, make it
     if (err.code === 'ENOENT') {
       await mkdir.mkdirs(dir)
-      await fs.writeFile(file, '')
+      await makeFile(file)
       return
     } else {
       throw err
@@ -29,7 +42,7 @@ async function createFile (file) {
   }
 
   if (dirStats.isDirectory()) {
-    await fs.writeFile(file, '')
+    await makeFile(file)
   } else {
     // parent is not a directory
     // This is just to cause an internal ENOTDIR error to be thrown